config/file_config.go

```diff
@@ -550,6 +550,22 @@ func (f *fileConfig) GetCollectorType() (string, error) {
 	return f.conf.Collector, nil
 }
 
+func (f *fileConfig) GetRunningConfig() (map[string]interface{}, error) {
+	configs := make(map[string]interface{})
+	keys := f.config.AllKeys()
+	for _, key := range keys {
+		configs[key] = f.config.GetString(key)
+		// mask sensitive strings
+		maskKeys := []string{"key", "password", "token"}
```

Comment:
Are we sure this is exhaustive?  Are there any free-form config options a user could use that might contain sensitive information?

Reply:
These are all the ones that will be unmarshalled by the file_config reader. 

I'm iffy on whether we want to bother masking them since the endpoint is authenticated and making sure a list of keys was properly updated may be a valid use case.

Reply:
If someone adds an arbitrary config line like

```toml
SecretFormula = "add cajun at the end"
```

Refinery errors out like this: 

```console
unable to load config: 1 error(s) decoding:

* '' has invalid keys: secretformula
Process 102407 has exited with status 1
```